### PR TITLE
Documentation and Today extension test

### PR DESCRIPTION
**Pull request by brockboland**
Not much actually changed here: I mimicked the Today extension that @chillpop added in #94 to confirm Vokoder works in extensions, and updated a bit of documentation. I didn't cherry-pick Carl's changes because the additional subspec isn't required now that `VOKMigrationFailureOptionWipeRecoveryAndAlert` has been removed (in #105).

### CODE REVIEW

**Comment by brockboland** [comment]
While Travis is chewing on this, anybody want to take a look? @chillpop @designatednerd @vokal/ios-developers

File: SampleProject/TodayWidget/TodayViewController.m [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+@end
+
+@implementation TodayViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    //Make sure the widget compiles and can access Vokoder
+    [[VOKCoreDataManager sharedInstance] setResource:@"VOKCoreDataModel" database:@"VOKCoreDataModel.sqlite"];
```
Comment (designatednerd) [inline SampleProject/TodayWidget/TodayViewController.m]: might be worth clarifiying that this is not accessing a shared container, but a core data database that is stored within the today extension.

Comment (brockboland) [inline SampleProject/TodayWidget/TodayViewController.m]: I'll push a commit for this once Travis has run. I don't want to lose the spot in their queue

**Comment by brockboland** [comment]
Also note: once this PR is merged, I'm planning to tag off 4.0.

**Comment by brockboland** [comment]
Sweet, Travis is happy. Anyone else want to take a look at this? Any objections to releasing 4.0?

**Comment by bryanluby** [comment]
LGTM

**Pull request merged**